.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,8 +10,6 @@ Request for comments document (if applies):
 
 ### Checklist ✅
 
-- [ ] The code architecture and patterns are consistent with the rest of the codebase.
-- [ ] The changes have been tested following the [documented guidelines](https://docs.tuist.io/contributors/testing-strategy/).
```

Comment:
This is enforced by the CI

Reply:
I think it's worth a reviewer manually testing / verifying the changes where possible. Automated tests are of course preferred, but sometimes a manual test can help spot some interesting things an automated test may have not been written for :)

Reply:
I agree, but:
- adding a checkbox wouldn't probably do much difference
- the link basically just says to run unit and integration/automation tests

Reply:
Maybe we can include the link as a comment at the very least? The link not only contains how to run tests but how we test in the codebase in general. I'd say it's good to nudge new contributors to test their PRs _and_ tell them how to test.